anagrafica/templates/anagrafica_elenchi_delegati.html

```diff
@@ -0,0 +1,47 @@
+{% extends 'us_elenchi_inc_vuoto.html' %}
```

Comment:
C'e' un motivo per questo template? Mi sembra non abbia nulla di differente all'elenco dei soci classico. Potrebbe estendere quel template... e basta.

In realta' pero' qui c'e' pure un problema che ha trovato @luca-dex in https://github.com/CroceRossaItaliana/jorvik/pull/405#issuecomment-270695314. In particolare, `{{ me.sede_riferimento }}` trova la principale sede di appartenenza (i.e. la sede dove il delegato e' registrato come volontario), che spesso non coincide alla sede dove si esercita la delega. Inoltre, potrebbero esservi piu' deleghe, presso diverse sedi, mentre l'appartenenza principale e' sempre una.